src/screens/profile/profile.js

```diff
@@ -0,0 +1,314 @@
+                                            <View style={{ paddingHorizontal: 10, marginBottom: 40 }}>
+                                                <Text style={{ fontSize: 16, fontWeight: 'bold', color: '#86939e', marginBottom: 10 }}>Genders</Text>
+                                                {values.gender ?
+                                                    <View>
+
+                                                        {
+                                                            AppConstants.GENDER.map((item) => {
+                                                                let element = values.gender.indexOf(item.id)
+                                                                console.log(element, item)
+                                                                if (element > -1) {
+                                                                    return (<Text>{item.name}</Text>)
+                                                                }
+                                                            })
+                                                        }
+                                                    </View>
+                                                    : null}
+                                            </View>
+                                        }
+                                    </View>
+                                    <View style={styles.formField}>
+                                        <Input
+                                            label={isEdit ? '' : 'DOB'}
+                                            inputContainerStyle={isEdit ? {} : { borderBottomWidth: 0 }}
+                                            placeholder={isEdit ? 'DOB (Date Month Year)' : ''}
+                                            value={values.dob}
+                                            onChangeText={handleChange('dob')}
+                                            errorMessage={errors.dob}
+                                            editable={isEdit}
+                                        />
+                                    </View>
+                                    <View style={styles.picker}>
+
+                                        {isEdit ?
+                                            <View>
+                                                <MultiSelect
+                                                    //   hideTags
+                                                    styleDropdownMenuSubsection={{ backgroundColor: 'transparent', borderBottomColor: '#86939e' }}
+                                                    styleTextDropdown={[{ color: '#86939e', fontSize: 17 }, isEdit ? null : { fontWeight: 'bold' }]}
+                                                    items={AppConstants.INTERESTS}
+                                                    uniqueKey="id"
+                                                    ref={(component) => { this.multiSelect = component }}
```

Comment:
I am seeing this huge component again. We should make a common component for it and reuse it at both places.